### DISCUSSION TITLE
添加 关闭推荐页顶部横版推广条 的选项

### DISCUSSION THREAD

**Discussion by sharkucn** [ywmoyue/biliuwp-lite#706] [feature]
### 问题描述

推广内容质量差/不合兴趣，希望首页（推荐页）更干净一点

### 描述解决方案

设置内添加一项，允许关闭该栏显示

### 附加信息

![image](https://github.com/user-attachments/assets/8eb013a1-1158-4a66-ab33-360c2370847a)

### 代码贡献

否

**Comment by zerosnowe** [comment]
一样，希望上面改成热门视频也不错
